app/src/main/res/values/strings.xml

```diff
@@ -714,4 +714,7 @@
     <string name="night_theme_summary">Select your favorite night theme — %s</string>
     <string name="select_night_theme_toast">You can select your favorite night theme below</string>
     <string name="download_has_started">Download has started</string>
+    <string name="tablet_mode_title">Tablet Mode</string>
+    <string name="on">On</string>
+    <string name="off">Off</string>
```

Comment:
These are translatable, which we almost definitely don't want them to be.

Reply:
Actually, ignore that. I thought you were using them in the Java code, but it turns out you're not.

Reply:
Are you sure? These are user-facing strings. For the non-userfacing key IDs (which should not be translatable and are marked as non-translatable) there are `@string/tablet_mode_auto_key`, `@string/tablet_mode_on_key` and `@string/tablet_mode_off_key`.

Reply:
No, I was mistaken. Ignore my comment.

Reply:
Sorry, Github doesn't auto-update, so I didn't see your second comment before posting. So please disregard my comment as well.